All Component/BlockUi.css

```diff
@@ -0,0 +1,511 @@
+html{
+    font-size: 62.5%;
+    padding: 0rem;
+    margin: 0rem;
+}
+
+body{
+    padding: 0rem;
+    margin: 0rem;
+}
+
+:root{
+    --background-color  : white;
+    --color-red : red;
+    box-sizing: border-box;
+    --color-secondary: #E75A0D;
+    --color-primary: #17183B;
+    --font-white : white;
+    --info-alert-background : rgb(207, 231, 238);
+    --info-alert-border : rgb(158, 222, 241);
+    --error-alert-background : rgb(229, 188, 188);
+    --erorr-alert-border : rgb(219, 148, 151);
+    --success-alert-background : rgb(177, 243, 186);
+    --success-alert-border : rgb(134, 236, 156);
+    --warning-alert-background : rgb(230, 231, 158);
+    --warning-alert-border : rgb(223, 240, 71);
+    --color-green : green;
+    --color-red : red;
+    --color-orange  :orange;
+    --background-color : white;
+}
+
+/* Card coponent css */
+.card-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.card{
+
+    background-color: white;
+    border: none;
+    margin-left:0 ;
+    border-radius: 5px;
+    height: fit-content;
+}
+
+
+
+.content{
+    display: flex;
+    margin-right:1rem ;
+    flex-wrap: wrap;
+}
+
+.action{
+    display: flex;
+    justify-content: space-between;
+    padding: 1rem 0rem;
+
+}
+
+.image{
+    border-top-right-radius: 1rem;
+
+}
+
+.title{
+    margin-left: 1rem;
+    padding : 1rem;
+    font-size: 1.5rem;
+}
+
+
+.action-btn-div{
+    display: flex;
+    gap: 1rem;
+    margin-left: 1rem;
+}
+
+.reaction-btn-div{
+    display : flex;
+    gap: 0.5rem;
+    margin-right:0.5rem;
+}
+
+.reaction-btn-div .reaction-btn{
+    border: none;
+    background-color: #f2f2f2;
```

Comment:
Use color variable if you plan to reuse this color.

Reply:
okay i'll change this

Reply:
adding a color variable for #f2f2f2